test/common/request_info/filter_state_impl_test.cc

```diff
@@ -43,20 +43,20 @@ class SimpleType : public DynamicMetadata::Object {
   int value_;
 };
 
-class DynamicMetadataImplTest : public testing::Test {
+class FilterStateImplTest : public testing::Test {
 public:
-  DynamicMetadataImplTest() { resetDynamicMetadata(); }
+  FilterStateImplTest() { resetDynamicMetadata(); }
 
-  void resetDynamicMetadata() { dynamic_metadata_ = std::make_unique<DynamicMetadataImpl>(); }
-  DynamicMetadata& dynamic_metadata() { return *dynamic_metadata_; }
+  void resetDynamicMetadata() { dynamic_metadata_ = std::make_unique<FilterStateImpl>(); }
```

Comment:
Rename these methods/fields as well?

Reply:
Done.  (Really sorry about missing all these, by the way--a lesson to be more creating in use of regular expressions.)